.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Excellent job on adding the test.md file to the .gitignone file. however, the file is still visible in the repository.
<img width="899" alt="Screen Shot 2022-02-08 at 23 29 10" src="https://user-images.githubusercontent.com/79058364/153079030-10fdfc6e-f087-4fd3-9f43-878026843461.png">

the reason for this is that you created the file and committed it before adding it to the .gitigone file. to undo this you can run this command `git rm --cached test.md`. then you can add and commit. with that the file will not be available in the repository. Let's go and fix it 😄

Reply:
Thank you @BenMukebo, It is such a constructive review.